jobs/integr8ly/browser-based-testsuite-pipeline.yaml

```diff
@@ -69,5 +70,16 @@
                         junit allowEmptyResults:true, testResults: '**/*.xml'
                     }
                 }
+                
+                stage('Publish screenshots for any failures/ errors') {
+                    dir('integreatly-qe/tests/ui-testsuite') {
+                        String testsuiteDirectories = sh returnStdout: true, script: 'ls'
+                        if (testsuiteDirectories.contains('screenshots')) {
```

Comment:
I think there are nicer ways to check if a file exists, see "Pipeline Syntax" link in job configuration. It is handy sometimes. Look for "fileExists" in the selector. Or look here:https://stackoverflow.com/questions/38534781/check-if-a-file-exists-in-jenkins-pipeline.

But this would work so I will approve.

Reply:
Thanks. I have changed the directory check as per your suggestion. It looks nicer and works: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/ppaszki-browser-based/10/ && https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/ppaszki-browser-based/11/

Reply:
@trepel can you have one more look please?